### PR TITLE
refactor minimap code

### PR DESCRIPTION
**Pull request by jmdejong**
iter_minimap_points now uses other existing methods
get_minimap_color added to decrease code duplication.

This is some cleanup that I originally wanted to include in PR #2786

### CODE REVIEW

**Comment by LinuxDonald** [comment]
@AndyMender could you review it?

**Comment by AndyMender** [comment]
@LinuxDonald later today or tomorrow, yes :).

File: horizons/gui/widgets/minimap.py [hunk 1]
```diff
@@ -34,70 +34,44 @@
 from horizons.util.shapes import Circle, Point, Rect
 
 
-def iter_minimap_points(location, world, island_color, water_color, area=None):
+def get_minimap_color(world_coords, world, island_color, water_color):
```
Comment (AndyMender) [inline horizons/gui/widgets/minimap.py]: Could you please add a docstring documenting what the function does specifically?
`iter_minimap_points_colors` is nicely documented, but `get_minimap_color `isn't.

**Comment by jmdejong** [comment]
Good point, I added a docstring. Do you think this is good enough or should I add more information?

**Comment by AndyMender** [comment]
@jmdejong in general it's a good practice to use the Sphinx-style docstring blocks for RestructuredText processing (:param:, :return:, etc.). This allows one to build HTML documentation from in-code docstrings with basically one click. However, this project has never been really documented that way so I think your docstring is sufficient :).

**Pull request merged**